src/components/atoms/ExpandableList/ExpandableList.tsx

```diff
@@ -43,7 +43,7 @@ export const ExpandableList = (props: ExpandableListProps) => {
       {renderItemList()}
       {items.length > itemLimit && (
         <Hide below="md">
-          <Button data-testid="verification-expand" onClick={handleExpand} variant="underlinedLink">
+          <Button data-testid="verification-expand" onClick={handleExpand} variant="underlinedLink" style={{ color: `#00242C`, fontWeight: 500 }}>
```

Comment:
could you reference the color value like `grey.500` in place of hardcoded hexcolor.

Reply:
Inez only provided the hexadecimal of the colour. The description she provided (midnight green) did not resolve to any colours readable by CSS.